src/main/resources/assets/totaleconomy/sql_refactor/planned_schema.sql

```diff
@@ -52,7 +52,7 @@ CREATE TABLE currencies (
 # Its current use is for foreign key consistency.
 # #
 CREATE TABLE jobs (
-  uuid         CHAR(36)    UNIQUE PRIMARY KEY
+  ident         CHAR(36)    UNIQUE PRIMARY KEY
```

Comment:
What's the reason for changing the column name to `ident`?

Reply:
See point 2 of "notes" ^^

Reply:
Haha sorry about that. Thanks.

Reply:
No worries 😉